_posts/2020-06-29-release-1.11.0.md

```diff
@@ -0,0 +1,299 @@
+
+**User-defined Metrics in Python UDFs ([FLIP-112](https://cwiki.apache.org/confluence/display/FLINK/FLIP-112%3A+Support+User-Defined+Metrics+in++Python+UDF))**
+
+To make it easier for users to monitor and debug the execution of Python UDFs, PyFlink now allows gathering and exposing metrics to external systems, as well as defining user scopes and variables. You can access the metrics system from a UDF by calling ``function_context.get_metric_group()`` in the open method, as described in the [documentation](https://ci.apache.org/projects/flink/flink-docs-master/dev/table/python/metrics.html#registering-metrics).
+
+<hr>
+
+## Important Changes
+
+ * [[FLINK-17339](https://jira.apache.org/jira/browse/FLINK-17339)] The Blink planner is the **default** in the Table API/SQL starting from Flink 1.11. This was already the case for the SQL Client since Flink 1.10. The old Flink planner is still supported, but not actively developed.
+
+ * [[FLINK-5763](https://issues.apache.org/jira/browse/FLINK-5763)] Savepoints now contain all their state inside a single directory (both metadata and program state). This makes it straightforward to figure out which files make up the state of a savepoint and allows users to **relocate savepoints** by simply moving a directory.
+
+ * [[FLINK-16408](https://issues.apache.org/jira/browse/FLINK-16408)] To reduce pressure on the JVM metaspace, the user code class loader is being reused by a ``TaskExecutor`` as long as there is at least a single slot allocated for the respective job. This changes Flink's recovery behaviour slightly, so that it will not reload static fields.
+
+* [[FLINK-11086](https://issues.apache.org/jira/browse/FLINK-11086)] Flink now supports Hadoop versions above **Hadoop 3.0.0**. Note that the Flink project does not provide any updated "flink-shaded-hadoop-\*" jars. Users need to provide Hadoop dependencies through the ``HADOOP_CLASSPATH`` environment variable (recommended) or the lib/ folder.
+
+* [[FLINK-16963](https://issues.apache.org/jira/browse/FLINK-16963)] All ``MetricReporters`` that come with Flink have been converted to plugins. These should no longer be placed into ``/lib`` (which may result in dependency conflicts), but ``/plugins/<some_directory>`` instead.
+
+* [[FLINK-12639](https://issues.apache.org/jira/browse/FLINK-12639)] The Flink **documentation** is undergoing some **rework**, so you might notice that the navigation and organization of content look slightly different starting from Flink 1.11.
+
+
+## Release Notes
+
+Please review the [release notes]({{ site.DOCS_BASE_URL }}flink-docs-release-1.11/release-notes/flink-1.11.html) carefully for a detailed list of changes and new features if you plan to upgrade your setup to Flink 1.11. This version is API-compatible with previous 1.x releases for APIs annotated with the @Public annotation.
```

Comment:
I would propose to add a link to the release notes to the section on the top where we mention the Downloads page, release changelog and updated docs.
Given Thomas experience during the release testing, it seems that this document is important for existing users!

Reply:
Yes, I already have this change locally. 👌